scripts/NuGetUtilities.psm1

```diff
@@ -39,13 +47,17 @@ function Get-PackageDirectoryName($configuration) {
 function New-NuGetPackageFromProjectFile($configuration, $project, $version) {
     $projectFile = "$SourceRoot\$project\$project.csproj"
 
+	# Note that we override the standard version construct via the VersionPrefix and 
+	# VersionSuffix properties so that we can specify an additional suffix that's only
+	# applied to the package.
     $arguments =
         "pack", $projectFile,
         "--configuration", $configuration,
         "--no-build", "--no-restore",
         "--include-source", "--include-symbols",
         "-p:Platform=$Platform",
-        "--output", (Get-PackageDirectoryName $configuration)
+        "--output", (Get-PackageDirectoryName $configuration),
+		"-p:Version=$version"
```

Comment:
>Version [](start = 6, length = 7)

If I'd done this the right way, wouldn't need to override the version property here. Oh well, didn't fully grok the world until I finished up (and need to move on). I'll put the hood up again later.